activerecord/lib/active_record/timestamp.rb

```diff
@@ -28,6 +28,10 @@ module ActiveRecord
   #
   #   ActiveRecord::Base.time_zone_aware_types = [:datetime]
   #
+  # NOTE: If you are using PostgreSQL you are likely to add <tt>tsrange</tt> and <tt>tstzrange</tt> to timezone aware types:
+  #
+  #   ActiveRecord::Base.time_zone_aware_types += [:tsrange, :tstzrange]
+  #
```

Comment:
May be we can add a generic note in configurations guide not just talking about Postgres specific types.